lib/net.js

```diff
@@ -513,6 +514,12 @@ Object.defineProperty(Socket.prototype, 'bufferSize', {
   }
 });
 
+Object.defineProperty(Socket.prototype, kUpdateTimer, {
+  get: function() {
```

Comment:
We should probably avoid getters for hot paths like a socket read callback function.

Reply:
I wouldn’t expect it to make a big difference? I’d expect V8 to inline this – if benchmark runs come back okay, is this still an issue?

That being said, it might be nice to move over to using `kUpdateTimer` in all cases.